app/models/content_item.rb

```diff
@@ -1,6 +1,8 @@
 class ContentItem < ApplicationRecord
   has_and_belongs_to_many :organisations
   has_and_belongs_to_many :taxonomies
+  has_one :audit, primary_key: :content_id, foreign_key: :content_id
```

Comment:
Do we need this association? Should content have visibility of whether it's been audited?

My mental model is that auditing is an entirely separate, standalone concept. By adding this association, it feels like we're coupling content with auditing. Happy to discuss.